src/gpu/intel/jit/gemm/selector/db/kernel.db

```diff
@@ -1165,5 +1165,15 @@ auto _CATALOG_ = kcatalog::toArray({
+{{'G', "gemm", {"[FO]", "O", "I"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {1, 1, 1}, "I"}, "aS32x2+S128@64 aB32+S128@208 aB wg 16x1 cb4 ks32 ql sys af vav hi sr br dm grf256", {16, (LoopType) 255, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {262144, 1048576, 16777216}, {262144, 1048576, 16777216}, {16, 64, 32}, {16, 1, 1}, 1, (WGType) 1, 257, 8192, 0, {1, 1, 4}, {true, true, true}}, {'E', 17, {363026, 218374, 0, 0, 0, 0, 0.427661, 0.921899, 3.22833, 8.31834, 0.00660424, 0.00724415, 0, 1, 1.11721, 0.967565, 1.89715e-12}}},
+{{'G', "gemm", {"[FO]", "O", "I"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {1, 1, 1}, "I"}, "aS32+S32@32 aB32+S1,128@48 aB wg 32x1 cb4x2 ks32 ql sys xaf rr fx vav hi sr br dm grf256", {16, (LoopType) 255, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {262144, 2097152, 16777216}, {262144, 2097152, 16777216}, {16, 128, 32}, {32, 1, 1}, 1, (WGType) 1, 257, 16384, 0, {1, 1, 4}, {true, true, true}}, {'E', 17, {136765, 426509, 0, 0, 0, 0, 0.58755, 1.39181, 3.57533, 8.73434, 0.00414952, 0.00414952, 0, 0.97197, 1.11401, 0.992638, 8.33384e-13}}},
+{{'G', "gemm", {"[FO]", "O", "I"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {1, 1, 1}, "I"}, "aS32x2+S32@64 aB32+S1,64@128 aB wg 32x1 cb3 ks32 ql nb 32x0 sys xaf rr fx vav hi sr br sm sn dm grf256", {16, (LoopType) 255, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {262144, 2097152, 16777216}, {262144, 2097152, 16777216}, {16, 128, 32}, {32, 1, 1}, 1, (WGType) 1, 257, 12288, 0, {1, 1, 4}, {true, true, true}}, {'E', 17, {140988, 386262, 0, 0, 0, 0, 0.631287, 1.49818, 3.60877, 8.74858, 0.00461733, 0.00461733, 0, 1, 1.10555, 0.996562, 7.63982e-13}}},
+{{'G', "gemm", {"[FO]", "O", "I"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {1, 1, 1}, "I"}, "aS32x2+S32@48 aB32+S64@96 aB wg 32x1 cb3x2 ks32 ql nb 32x0 sys xaf rr fx vav hi sr br sm sn dm grf256", {16, (LoopType) 255, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {262144, 1048576, 16777216}, {262144, 1048576, 16777216}, {16, 64, 32}, {32, 1, 1}, 1, (WGType) 1, 257, 6144, 0, {1, 1, 4}, {true, true, true}}, {'E', 17, {127181, 347287, 0, 0, 0, 0, 0.533741, 1.64384, 3.35075, 8.49343, 0.00665864, 0.00665864, 0, 1, 1.08743, 0.995661, 6.75068e-13}}},
+{{'G', "gemm", {"[FO]", "O", "I"}, {"T", "N", "N"}}, {-1, -1, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {-1, -1, -1}, {1, 1, 1}, "I"}, "aS32x2+S96@144 aB32x2+S96@112 aB wg 16x1 cb3 ks64 ql nb 16x0 sys xaf st vav hi sr br dm grf256", {16, (LoopType) 255, 256, {(LoopType) 144, (LoopType) 255, (LoopType) 255}, {262144, 524288, 16777216}, {262144, 524288, 16777216}, {16, 32, 64}, {16, 1, 1}, 1, (WGType) 1, 257, 6144, 0, {1, 1, 4}, {true, true, true}}, {'E', 17, {302330, 158230, 0, 0, 0, 0, 0.400286, 1.11464, 2.9277, 7.92924, 0.0116958, 0.00885108, 0.00204323, 1, 1.06933, 0.947726, 2.43405e-12}}}
```

Comment:
What is the motivation for backporting to rls-v3.9 as this does not look like a bug-fix?